tests/minimal_webgl/CMakeLists.txt

```diff
@@ -7,6 +7,8 @@ endif()
 
 project(minimal_webgl)
 
+option(MEMORY64 "build for 64-bit Wasm" OFF)
```

Comment:
This test is meant to focus on WebGL specifically, and also to be an example of a minimal program that we refer people to. So I'd prefer to not add things to it that are for other purposes.

We have more minimal tests that might be better to start with, like maybe adding this?
```python
  def test_memory64_hello_world(self):
    self.set_setting('MEMORY64', 1)
    self.do_run_in_out_file_test('tests', 'core', 'test_hello_world.c')
```